README.md

```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://gitlab.parity.io/parity/rust-secp256k1/badges/master/build.svg)](https://gitlab.parity.io/parity/rust-secp256k1/commits/master)
-![crates.io](https://img.shields.io/crates/v/parity-secp256k1.svg)
+[![crates.io](https://img.shields.io/crates/v/parity-secp256k1.svg)](https://crates.io/crates/parity-secp256k1)
```

Comment:
This fixes a link to a badge for `parity-secp256k1`, but you opened this PR to `rust-secp256k1`. I assume this was just a mistake and you wanted to open a PR to `parity-secp256k1`. This seems is also the reason for the large diff including 87 commits before this one. I'll close this oone

Reply:
> This seems is also the reason for the large diff including 87 commits before this one. I'll close this one.

Indeed, tooling failure on my end. I apologize for the noise. :/